src/scripts/components/SlidingRadio.jsx

```diff
@@ -0,0 +1,91 @@
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import React, { Component, Fragment } from 'react';
+
+import { UtilitySystem, FormLabel } from '.';
+
+class SlidingRadio extends Component {
+  state={
+    selectedValue: this.props.selectedValue || '',
+  }
+
+  handleChange = (value) => {
+    this.setState({ selectedValue: value });
+    if (this.props.onChange) this.props.onChange(value);
+  }
+
+  getLabelStyle = (value) => {
+    const isChecked = value === this.state.selectedValue;
+    const { options } = this.props;
+
+    let style;
+    if (!isChecked) {
```

Comment:
I noted this above, but let's consider making these dynamic based on what is included in the option prop. I'd expect to be able to pass an option that includes both the value as well as the color class I want that option to represent.

Reply:
👍